test/helpers/auth_source_external_helper_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class AuthSourceExternalHelperTest < ActionView::TestCase
+  include AuthSourceExternalHelper
+  include TaxonomyHelper
+
+  describe 'tab_classes_for_edit_auth_source_external' do
+    context 'user with view_locations and view_organizations permission' do
+      it 'returns class for location tab' do
+        as_admin do
+          assert_equal ({ :location => 'active' }), tab_classes_for_edit_auth_source_external
+        end
+      end
+    end
+
+    context 'user with view_locations permission only' do
+      it 'returns class for location tab' do
+        setup_user 'view', 'locations'
+        assert_equal ({ :location => 'active' }), tab_classes_for_edit_auth_source_external
```

Comment:
```suggestion
       assert_not_empty tab_classes_for_edit_auth_source_external[:location]
```

Reply:
@kgaikwad The suggested changes will not cover the method's functionality completely.
We are not only testing here that location tab has 'active' classes, but also that organization tab does not have any 'active' class. 
`assert_not_empty tab_classes_for_edit_auth_source_external[:location]` only checks that :location tab is active. It does not check that the :organization key is not present.

Reply:
> It does not check that the :organization key is not present.

Suggested this change because for any change in tab_classes, one will need to touch these test cases unnecessarily. What do you think about adding multiple assert statements in one test? Here, for example -

```
assert_equal [:location], tab_classes_for_edit_auth_source_external.keys
assert_not_empty tab_classes_for_edit_auth_source_external[:location]
```